gabarit/template_num/num_project/tests/test_utils_models.py

```diff
@@ -693,7 +693,7 @@ def test11_predict_with_proba(self):
         self.assertEqual(sum([round(p[0]) for p in proba]), sum([len(_) for _ in y_test_classification_multi]))
         pred, proba = utils_models.predict_with_proba(x_test_2_solo, model)
         self.assertEqual(pred, y_test_2_classification_multi_solo)
-        self.assertTrue(proba[0] >= 0.5)
+        self.assertTrue(proba[0][0] >= 0.5)
         remove_dir(model_dir)
 
         ################
```

Comment:
Add test with several inputs

Reply:
this is the test for mono input, multi output is just above